embuilder.py

```diff
@@ -313,6 +314,10 @@ def is_flag(arg):
         }
 
       ''', [libname('libpthreads_stub')])
+    elif what == 'ubsan-rt-wasm':
+      if not shared.Settings.WASM_BACKEND:
+        continue
+      build(C_BARE, ['libsanitizer_common_rt_wasm.a', 'libubsan_rt_wasm.a'], ['-fsanitize=undefined', '-s', 'WASM=1'])
```

Comment:
It looks like you're building the sanitizer code with ubsan enabled. Is that necessary? Shouldn't the sanitizer code be trusted to run correctly?